ml_metadata/tools/mlmd_bench/fill_types_workload_test.cc

```diff
@@ -23,59 +25,431 @@ limitations under the License.
 #include "ml_metadata/proto/metadata_store.pb.h"
 #include "ml_metadata/proto/metadata_store_service.pb.h"
 #include "ml_metadata/tools/mlmd_bench/proto/mlmd_bench.pb.h"
+#include "ml_metadata/tools/mlmd_bench/util.h"
 #include "tensorflow/core/lib/core/status_test_util.h"
 
 namespace ml_metadata {
 namespace {
 
-constexpr int64 kNumOperations = 100;
+constexpr int kNumberOfOperations = 100;
+constexpr int kNumberOfExisitedTypeInInsert = 80;
+constexpr int kNumberOfExistedTypesButNotEnoughInUpdate = 50;
+constexpr int kNumberOfExistedTypesEnoughInUpdate = 200;
+
+// Enumerates the workload configurations as the test parameters that ensure
+// test coverage.
+std::vector<WorkloadConfig> EnumerateConfigs(const bool is_update) {
+  std::vector<WorkloadConfig> config_vector;
+  WorkloadConfig template_config = testing::ParseTextProtoOrDie<WorkloadConfig>(
+      R"(
+        fill_types_config: { num_properties { minimum: 1 maximum: 10 } }
+      )");
+
+  template_config.set_num_operations(kNumberOfOperations);
+  template_config.mutable_fill_types_config()->set_update(is_update);
+  template_config.mutable_fill_types_config()->set_specification(
+      FillTypesConfig::ARTIFACT_TYPE);
+  config_vector.push_back(template_config);
+  template_config.mutable_fill_types_config()->set_specification(
+      FillTypesConfig::EXECUTION_TYPE);
+  config_vector.push_back(template_config);
+  template_config.mutable_fill_types_config()->set_specification(
+      FillTypesConfig::CONTEXT_TYPE);
+  config_vector.push_back(template_config);
+
+  return config_vector;
+}
+
+// Inserts some types into db to set it up in different start status in
+// testing.
+tensorflow::Status InsertTypesInDb(const int64 num_artifact_types_to_insert,
+                                   const int64 num_execution_types_to_insert,
+                                   const int64 num_context_types_to_insert,
+                                   MetadataStore* store) {
```

Comment:
ditto for these names, `num_artifacts_types`, as the function name describes the purpose.

Reply:
Noted! Sounds good!